### PR TITLE
Ankit | Test | Prod - On create/update voucher flow, discount value is displaying with many digits after decimal.

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwu183q

### CODE REVIEW

File: apps/web-giddh/src/app/services/general.service.ts [hunk 1]
```diff
@@ -2149,6 +2149,20 @@ export class GeneralService {
         };
     }
 
+    /**
+     * Round a Number to Company Decimal Places
+     *
+     *
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/services/general.service.ts]:
```suggestion
```

File: apps/web-giddh/src/app/services/general.service.ts [hunk 2]
```diff
@@ -2149,6 +2149,20 @@ export class GeneralService {
         };
     }
 
+    /**
+     * Round a Number to Company Decimal Places
+     *
+     * 
+     * @param {number} value
+     * @param {number} companyDecimalPlaces
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/services/general.service.ts]:
```suggestion
     * @param {number} [companyDecimalPlaces=2]
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/services/general.service.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/services/general.service.ts]: Done

File: apps/web-giddh/src/app/services/general.service.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/services/general.service.ts]: Done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**